### PR TITLE
Updates for ITKv5.2 code cleanups ... ROUND1

### PR DESCRIPTION
**Pull request by hjmjohnson**

### CODE REVIEW

File: include/itkVariationalRegistrationFastNCCFunction.hxx [hunk 1]
```diff
@@ -19,7 +19,7 @@
 #define itkVariationalRegistrationFastNCCFunction_hxx
 
 #include "itkVariationalRegistrationFastNCCFunction.h"
-#include "itkExceptionObject.h"
+#include "itkMacro.h"
```
Comment (dzenanz) [inline include/itkVariationalRegistrationFastNCCFunction.hxx:22]: Misleading commit message

**Review by dzenanz — APPROVED** [review]

**Comment by hjmjohnson** [comment]
@jhlegarreta FYI: Please look at the last patch https://github.com/InsightSoftwareConsortium/ITKVariationalRegistration/pull/40/commits/39acd985bf86534d037a68a17f729791f70efe8e

**Comment by jhlegarreta** [comment]
@hjmjohnson Reading the error message your editions look fair, but would not be able to confirm the expected behavior.

I would remove the commented lines from the `wrapping` files. Not sure if you added them to give them a try.

The `Evaluate template` step failure the CI pipelines are reporting is somehow cryptic. Not sure what's the source of the issue.

**Comment by hjmjohnson** [comment]
@jhlegarreta I used code that matched code in ITK wrapping for similar objects.  In all cases, ITK also had the commented out wrapping code too, so I kept it the same here.

I'm working on the CI, but will not try to make it work 100%.  I only have time to make it better than it was.

**Comment by jhlegarreta** [comment]
OK, thanks for the explanation Hans.

Again :100: to the effort you are making on the remotes.

**Pull request merged**